dnf-docker-test/features/steps/test_behave.py

```diff
@@ -64,6 +65,14 @@ def diff_package_lists(a, b):
         map(_right_decorator, list(sb - sa)))
 
 
+def diff_query_lists(sack_a, sack_b):
```

Comment:
It compare sack created before dnf command with sack after dnf command. It returns packades, that were changed ir added or removed.